src/main/java/io/nats/streaming/SubscriptionOptions.java

```diff
@@ -191,7 +206,8 @@ public int hashCode() {
         result = 31 * result + (ackWait != null ? ackWait.hashCode() : 0);
         result = 31 * result + (startAt != null ? startAt.hashCode() : 0);
         result = 31 * result + (int) (startSequence ^ (startSequence >>> 32));
-        result = 31 * result + (startTime != null ? startTime.hashCode() : 0);
+        result = 31 * result + (startTime != null ? startTime.getNano() : 0);
```

Comment:
Just curious - why override hashCode?  For performance maybe save off the value after first build so it doesn't need to be rebuilt every time it's called.

Reply:
this is one of those places where i am trying to minimize the changes for this release. It was overridden before, so I just added the minimum to remain correct. I suspect there is little if any value in this as the options aren't used as keys anywhere.